sdk/core/core/inc/az_http_policy.h

```diff
@@ -28,6 +28,18 @@
 
 typedef struct az_http_policy az_http_policy;
 
+/**
+ * @brief options for retry policy
+ * max_retry = maximun number of retry intents before returning error
+ * delay_in_ms = waiting time before retrying in miliseconds
+ *
+ */
+typedef struct {
+  bool header;
```

Comment:
You MUST comment what this means: 
// true indicates that the api version is added as a header and false means it is added as a query parameter

Reply:
Fixed